modules/local/goleft/indexcov/main.nf

```diff
@@ -0,0 +1,51 @@
+process GOLEFT_INDEXCOV {
+label 'process_single'
+
+container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/goleft:0.2.4--h9ee0642_1':
+        'biocontainers/goleft:0.2.4--h9ee0642_1' }"
+
+input:
+        val(meta)
+        path(bams)
```

Comment:
is the tool run on all bams for all samples in one job?

Reply:
nevermind I see it further down. The docs say 30sec per 30 WGS 🤯 . Any idea how it scales with number of genomes?